.github/workflows/tutorials.yml

```diff
@@ -0,0 +1,95 @@
+name: Tutorials
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+  # cron for every two weeks
+  schedule:
+    - cron: "0 0 * * 1"
+
+jobs:
+  hello-world-contract-unix:
+    name: Hello World Contract Tutorial Unix
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Install Node.js (LTS)
+        uses: actions/setup-node@v3
+        with:
+          node-version: "lts/*"
+
+      - name: Install yarn
+        run: npm install -g yarn
+
+      - name: Install patch-package globally
+        run: npm install -g patch-package
+
+      - name: Install zksync-cli globally
+        run: npm install -g zksync-cli
+
+      - name: Create new project called greeter-example
+        run: echo | zksync-cli create-project greeter-example
+
+      - name: Add private key to WALLET_PRIVATE_KEY variable in .env.example file
+        run: |
+          if [[ "$RUNNER_OS" == "macOS" ]]; then
+            cd greeter-example && sed -i '' 's/WALLET_PRIVATE_KEY=/WALLET_PRIVATE_KEY=ac7de7276038f2c181c541331f38d5913a8d63414752295e0011b3a286387043/' .env.example
+          else
+            cd greeter-example && sed -i 's/WALLET_PRIVATE_KEY=/WALLET_PRIVATE_KEY=ac7de7276038f2c181c541331f38d5913a8d63414752295e0011b3a286387043/' .env.example
+          fi
+
+      - name: Rename .env.example to .env
+        run: cd greeter-example && mv .env.example .env
+
+      - name: Compile contracts with hardhat
+        run: cd greeter-example && yarn hardhat compile
+
+      - name: Deploy contract
+        run: cd greeter-example && yarn hardhat deploy-zksync --script deploy-greeter.ts
```

Comment:
This doesn't test the hello world example, rather just deploys the greeter.sol contract from the hardhat-template? If anything this should live in the template repo and not here.

Reply:
Hm true. I'm about to propose moving the templates to the `zksync-cli` repo on my next PR to it, so perhaps this should even live there instead. But then again this tests the sequence of commands present in the docs, so maybe even this CI should be added to the docs repo instead. Wdyt?

Reply:
Why not? This would make the maintainers of the cli clearly responsible for working templates

Reply:
Regardless of where the templates live, the ci check for deploying the greeter.sol contract from the hardhat-template should not live here but rather where the actual implementation is.